lua/a-star.lua

```diff
@@ -77,18 +78,23 @@ function pos_eq(a,b)
 	return a.x == b.x and a.y == b.y
 end
 
-
 function a_star(start, goal)
-	open_set_q = priorityqueue()
-	open_set_tb = {}
+	--print("Running our path")
```

Comment:
Consider either removing or commenting all prints if we need them for future debug purposes